src/en/charms-deploying.md

```diff
@@ -1,42 +1,37 @@
 Title: Deploying applications
 TODO: Add 'centos' and 'windows' stuff to series talk
-      Review whether Juju should go to the store when pointing to a local dir
-        with non-existant charm. It did not for me but the old version of this
-        doc said it should.
-      Review required. Channnels especially
-      This page is too long. It should contain just basic stuff and link to sub-pages.
       Hardcoded: Ubuntu codenames
-      Verify MAAS spaces example
-      Bug tracking: https://bugs.launchpad.net/juju/+bug/1747998
+      Move commented section to a troubleshooting page
 
 # Deploying applications
 
-The fundamental point of Juju is that you can use it to deploy applications
-through the use of charms (the magic bits of code that make things just work).
-These charms can exist in the [Charm Store](https://jujucharms.com/store) or on
-the file system (previously downloaded from the store or written locally).
+The fundamental purpose of Juju is to deploy and manage software applications
+in a way that is fast and easy. All this is done with the help of *charms*,
+which are bits of code that contain all the necessary intelligence to do these
+things. Charms can exist online (in the [Charm Store][charm-store]) or on your
+local filesystem (previously downloaded from the store or written locally).
 
 Charms use the concept of *series* analogous as to how Juju does with Ubuntu
```

Comment:
In this new context, I think this paragraph needs rewriting too.

Reply:
:ok: